README.md

```diff
@@ -1,5 +1,29 @@
-# Momento client-types-php
+# client-sdk-php
 
-:warning: Pre-Alpha :warning:
+:warning: Experimental SDK :warning:
+
+PHP SDK for Momento is experimental and under active development. There could be non-backward compatible changes or
+removal in the future. Please be aware that you may need to update your source code with the current version of the SDK
+when its version gets upgraded.
+---
+
+<br />
+PHP SDK for Momento, a serverless cache that automatically scales without any of the operational overhead required by
+traditional caching solutions.
+
+<br/>
+
+# Getting Started :running:
+
+## Requirements
+
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+- At least PHP 7
+- The grpc PHP extension. See the [gRPC docs](https://github.com/grpc/grpc/blob/v1.46.3/src/php/README.md) section on installing the extension.
+
+## Using Momento
+
+Run `composer update` from the root of the repository to install the necessary prerequisites.
+
+Then check out full working code in `example.php` at the root of this repository!
```

Comment:
Probably need add section here for updating local protos for now.  Similar to https://github.com/momentohq/client-sdk-go#updating-grpc-protos

Reply:
The protos are bundled in the SDK for right now, so that doesn't belong in the README at this point.

Reply:
I mean more like dev doc for like how to regenerate/update if need to say if we add new API. Doesn't need to be in README maybe contributing.md? Just from perspective of like if I had to come in and bump these as someone contributing to this SDK.

Reply:
That process is in such flux (and only available to Momento internal) at this point that I'd rather put this off until we have something reasonable in place. Right now there's a `client-types-php` repo (that is going away in favor of a better solution when I have one) that has a manually triggered github action that produces a zipfile that you need to overwrite the protos in the SDK repository. Documenting the existing process isn't useful to the public at this point, but I'll definitely add a section for that when there's a sane process to document.